.github/workflows/ngsa.yaml

```diff
@@ -0,0 +1,50 @@
+name: Build NGSA-App
+
+on:
+  schedule:
+    # Run daily at 6:00 AM UTC
+    - cron: "0 6 * * *"
```

Comment:
Why do a build each morning?  I remember it being mentioned the other day, but I wasn't sure why we were doing that in addition to on push.